gitlab.go

```diff
@@ -137,7 +153,7 @@ func (l *NotificationLevelValue) UnmarshalJSON(data []byte) error {
 
 // List of valid notification levels.
 const (
-	DisabledNotificationLevel NotificationLevelValue = iota
+	DisabledNotificationLevel      NotificationLevelValue = iota
```

Comment:
You didn't run `gofmt` or `goimports`, I think you should really tweak your editor so that this always runs on save. Then you can just forget about it...

Reply:
hm - strange... for me `go fmt` removes this whitespace, since I didn't touch this line at all manually. Besides: thx for the hint with the auto-save, I configured that now

Reply:
But that is correct... The whitespace shouldn't be there and was added in your commit.